src/openforms/js/components/admin/form_design/variables/VariablesEditor.stories.js

```diff
@@ -181,18 +228,22 @@ export const WithObjectsAPIRegistrationBackends = {
   parameters: {
     msw: {
       handlers: [
-        mockTargetPathsPost([
-          {
-            targetPath: ['path', 'to.the', 'target'],
-            isRequired: true,
-            jsonSchema: {type: 'string'},
-          },
-          {
-            targetPath: ['other', 'path'],
-            isRequired: false,
-            jsonSchema: {type: 'object', properties: {a: {type: 'string'}}, required: ['a']},
-          },
-        ]),
+        mockTargetPathsPost({
```

Comment:
@Viicos I can't really judge if this still accurately reflects what the real API endpoint does, can you confirm this is okay?

Reply:
It should, I also updated the `mockTargetPathsPost` so that it returns the schema for the requested type (but the structure of the returned data is unchanged).

Reply:
the old code returned an array of two items, while the new variant grabs only a single item based on the request body. I think the new code is more correct in how the backend actually works, but I'm not well enough aware of the implementation details to be certain :grimacing:

Reply:
Yeah it is reflecting the backend better, as the available target paths depend on the JSON Schema type of the variable :)